src/scripts/components/App.js

```diff
@@ -21,7 +21,7 @@ const App = () => {
       <div>
         <Switch>
           <Route exact path="/" component={Home} />
-          <Route exact path="/pageLink" component={AsyncDynamicPage} />
+          <Route exact path="/link" component={AsyncDynamicPage} />
```

Comment:
if I try to access `localhost/link` directly, results in error `cannot /GET link`.  Is this a react-router-dom thing I just need to dig deeper into or a problem with server trying to find "/link"?